packages/console-client/README.md

```diff
@@ -0,0 +1,59 @@
+# Console-client
+
```

Comment:
```suggestion

> [!IMPORTANT]  
> Console Client Package is experimental, it may be subject to breaking changes and features only a minimal subset of APIs

```